kernel/device.c

```diff
@@ -84,17 +88,22 @@ const struct device *z_impl_device_get_binding(const char *name)
 	 * with CONFIG_* macros), only cheap pointer comparisons will be
 	 * performed. Reserve string comparisons for a fallback.
 	 */
+	LOG_DBG("device_get_binding(%s)", name);
 	for (dev = __device_start; dev != __device_end; dev++) {
 		if (z_device_ready(dev) && (dev->name == name)) {
+			LOG_DBG("   - found");
 			return dev;
 		}
 	}
 
 	for (dev = __device_start; dev != __device_end; dev++) {
+		LOG_DBG("   - %s, ready=%d", dev->name, z_device_ready(dev));
 		if (z_device_ready(dev) && (strcmp(name, dev->name) == 0)) {
+			LOG_DBG("   - found");
 			return dev;
 		}
 	}
+	LOG_DBG("   - not found");
```

Comment:
`LOG_DBG("%s not found", name);`

Reply:
done

Reply:
Can you change this to `LOG_DBG("%s not found", name);`

Reply:
done